earth_enterprise/tutorial/SConstruct

```diff
@@ -34,30 +34,62 @@ if (not installdir) and ('install' in COMMAND_LINE_TARGETS):
 tutorials_dir = installdir + '/tutorial'
 root_dir = tutorials_dir + '/opt/google/share/tutorials/fusion/'
 
-def makeHardLinks(instdir, target_dir, excluded_list):
-    # get contents of target_dir and make hard links to files in instdir
+# Get the current directory, since SCons will move into the directory
+#   it's using, and FusionTutorial is expected below the SConscript file.
+tutorial_src_dir = os.sep.join((os.getcwd().split(os.sep) +
+                    ['FusionTutorial']))
+
+def makeHardLinks(install_dir, target_dir, excluded_list):
+    # get contents of target_dir and make hard links to files in install_dir
     # recurse down the directories
 
     contents = os.listdir(target_dir)
-    target_dir = target_dir + "/"
+    target_dir = target_dir + "/"	
     # Ensure the install dir exists at each step
-    commands.getstatusoutput("mkdir -p " + instdir)
+    commands.getstatusoutput("mkdir -p " + install_dir)
     for file in contents:
         if file in excluded_list:
             continue
         target_file = target_dir + file
         if os.path.isdir(target_file):
-            subdir = instdir + "/" + file
+            subdir = install_dir + "/" + file
             makeHardLinks(subdir, target_file,
                                              excluded_list)
         else:
             commands.getstatusoutput("ln \"" + target_file + "\" " +
-                                     instdir)
+                                     install_dir)
+
+
+def makeSoftLinks(dest_dir, src_dir, excluded_list):
```

Comment:
There's a fair amount of duplication between this function and makeHardLinks. Is there a reasonable way to combine them? Alternatively, I'd be fine with removing makeHardLinks if it's no longer used. We have it in the git history if we ever need it again.

Reply:
I believe the simplest way solution is to simply remove makeHardLinks. This would answer your concern, @tst-rwildes's concerns, and my original question about the matter.

Reply:
Works for me.

Reply:
Done.